scss/_variables.scss

```diff
@@ -342,9 +342,9 @@ $typefaces: "display-1", "display-2", "display-3", "display-4", "h1", "h2", "h3"
   "body", "text-data", "headline", "subhead", "footnote", "caption", "overtext",
   "mono", "small", "bold", "heavy" !default;
 
-$icon-size: 1.25em !default;
-$icon-size-lg: 1.75em !default;
-$icon-size-sm: 0.75em !default;
+$icon-size: 1.42em !default;
```

Comment:
Will these styles affect icons on the web app? Or do we only need these for the docs section?

Reply:
These needs to be moved to docs, now that I know it existes